server.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import path from 'path';
+
+const port = process.env.PORT || 8080;
+
+const app = express();
+
+app.use(express.static(path.join(__dirname)));
+app.all('*', (req, res) => {
+  res.sendFile(path.join(__dirname, 'index.html'));
+});
+
+app.listen(port);
```

Comment:
I feel there should be feedback on the console/terminal indicating if the server started successfully or not.
```suggestion
app.listen(port, (error) => {
  if (error) {
    console.log(error);
  } else {
    console.log(`Sever is up and running on port ${PORT}`);
  }
});
```

Reply:
ESlint will flag console.log. Moreover, webpack will report the status of the build during development and Heroku logs details of requests to the server.